launch_ros/launch_ros/default_launch_description.py

```diff
@@ -61,7 +62,9 @@ def _function(self, context: launch.LaunchContext):
             if 'rcl_init called while already initialized' in str(exc):
                 pass
             raise
-        self.__launch_ros_node = rclpy.create_node('launch_ros', context=self.__rclpy_context)
+        self.__launch_ros_node = rclpy.create_node(
+            'launch_ros_{}'.format(os.getpid()),
```

Comment:
@BMarchi appending `launch` PID helps disambiguate but doesn't make it a hidden a node -- you need a leading underscore.

Reply:
I didn't see the requirement to make the launch_ros node hidden. Are you sure about appending that?

Reply:
Oops, I just tossed in a feature request. It's not in the original ticket, you're right. Disregard (I'll propose it afterwards).